faker.go

```diff
@@ -9,9 +9,14 @@ import (
 	"time"
 )
 
+// RandomGenerator interface.
+type RandomGenerator interface {
+	Int() int
+}
+
 // Faker is the Generator struct for Faker
 type Faker struct {
-	Generator *rand.Rand
```

Comment:
I want just to understand this change, if it is possible to use `faker.NewWithSeed()`, why adding this option?